bench/Main.hs

```diff
@@ -0,0 +1,20 @@
+module Main where
+
+import           Test.Tasty.Bench
+
+import           Bench.Database.LMDB.Simple.Cursor as Cursor
+
+main :: IO ()
+main = defaultMain [
+    bgroup "bench" [
```

Comment:
Why do we have "bench\Bench\..."?

Reply:
Oops :)

Reply:
My usual convention is to first list the name of the benchmark/test executable. In this case, it is a coincidence that the names are similar. Would you be okay with leaving it as is?